Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1951,9 +1951,9 @@ bool CStaticFunctionDefinitions::ShowPlayerHudComponent(eHudComponent component,
     return true;
 }
 
-bool CStaticFunctionDefinitions::IsPlayerHudComponentVisible(eHudComponent component, bool& bOutIsVisible)
+bool CStaticFunctionDefinitions::IsPlayerHudComponentVisible(eHudComponent component, bool bOutIsEnabled, bool& bOutIsVisible)
 {
-    bOutIsVisible = g_pGame->GetHud()->IsComponentVisible(component);
+    bOutIsVisible = g_pGame->GetHud()->IsComponentVisible(component, bOutIsEnabled);
     return true;
 }
```

Comment:
Since you're modifying this function, it's probably better to remove it. We don't put code in `CStaticFunctionDefinitions` anymore. You can call `g_pGame->GetHud()->IsComponentVisible` directly. The rest of the PR looks okay.